apps/src/lib/node/ledger/shell/finalize_block.rs

```diff
@@ -168,6 +168,7 @@ where
                         }
                         Event::new_tx_event(&tx_type, height.0)
                     }
+                    ProtocolTxType::BridgePoolVext(_) => continue,
```

Comment:
nit: maybe sprinkle a `tracing::debug!()` log line here, for debugging purposes. something like "skipping bridge pool vext"

Reply:
I'm literally going to fill this out in the next PR. This is just a placeholder.